docs/pages/account/deleteAccount.mdx

```diff
@@ -0,0 +1,23 @@
+import { Callout } from 'nextra/components'
+import { Steps } from 'nextra/components'
+
+# Mein Konto löschen
+
+<Callout type="warning">
+  Dieser Schritt wird sofort ausgeführt und kann nicht rückgängig gemacht
+  werden!
+</Callout>
+
+Falls du dein Konto und somit alle deine Daten löschen möchtest, kannst du dies
+über die Einstellungen machen. Führe dazu folgende Schritte aus:
+
+<Steps>
+### Einstellungen in Sharezone öffnen
+### "Mein Konto" öffnen
+### "Konto löschen" auswählen (unten auf der Seite)
+</Steps>
+
+<Callout type="info">
+Solltest du dich anonym angemeldet haben, wird dein Konto ebenfalls gelöscht,
+wenn du dich abmeldest.
+</Callout>
```

Comment:
We already have a docs page for deleting the account (see https://docs.sharezone.net/account/delete-account). Therefore, the `deleteAccount.mdx` can be deleted.